docs/advanced_usage.rst

```diff
@@ -404,3 +418,52 @@ running Cookiecutter on a template that requires custom Jinja2 extensions.
 .. _`Jinja2 extensions`: http://jinja2.readthedocs.io/en/latest/extensions.html#extensions
 .. _`now`: https://github.com/hackebrot/jinja2-time#now-tag
 .. _`jinja2_time.TimeExtension`: https://github.com/hackebrot/jinja2-time
+
+.. _`running hooks in place`:
+
+Running hooks in place and consuming serialized context
+-------------------------------------------------------
+
+Disabling hook duplication
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+As explained in `Using Pre/Post-Generate Hooks (0.7.0+)`_, you can disable the default behaviour of hook duplication. This is interesting when you don't need to use template variables directly in your hook.
+
```

Comment:
I think it's also important to note that you cannot use template variables in hook scripts if you're running in-place (and you would therefore __have__ to use `stdin` context if needed)